get_started.md

```diff
@@ -2,42 +2,59 @@
 
 The Gaia Sandbox provides a containerized simulation environment that allows you to experiment with samples written using the Gaia Platform.
 
-This sample models a factory that contains the following:
+This sample models a security access control to allow or deny people entering an office campus. It contains:
 
-TODO
+- A headquarters (HQ) building
+- Rooms inside the HQ building
+- People, of varying security access levels
+    - John: an employee with access to all rooms
+    - Jane: a visitor with limited room access
+    - A stranger: they are allowed inside the HQ building!
```

Comment:
Are they allowed in? Or is it that they aren't allowed in?

Reply:
Not allowed. That's a typo.